yb-voyager/cmd/conflictDetectionCache.go

```diff
@@ -107,10 +159,12 @@ func (c *ConflictDetectionCache) eventsConfict(event1, event2 *tgtdb.Event) bool
 	}
 	uniqueKeyColumns := c.tableToUniqueKeyColumns[maybeQualifiedName]
 	for _, column := range uniqueKeyColumns {
-		// if the unique key column value is same, then we have a conflict
-		if *event1.Fields[column] == *event2.Fields[column] {
+		if event1.BeforeFields[column] == nil || event2.Fields[column] == nil {
```

Comment:
event1 and event2 names are a bit confusing. 
How about `cachedEvent` (which you already use), and `incomingEvent`?

Reply:
yeah good idea